packages/core/src/storages/access_checking.ts

```diff
@@ -1,6 +1,6 @@
 import { AsyncLocalStorage } from 'async_hooks';
 
-const storage = new AsyncLocalStorage<{ checkFunction:() => void }>();
+const storage = new AsyncLocalStorage<{ checkFunction: () => void }>();
```

Comment:
lol, eslint being wrong. its really about the time to drop those formatting rules completely...